http/src/main/resources/swagger/api-docs.yaml

```diff
@@ -4031,6 +4031,11 @@ components:
         sourceWorkspaceId:
           type: string
           description: Optional id of source workspace if app is a clone.
+        autoDeleteThresholdDuration:
+          type: string
+          description: The duration of idle time (i.e. since last time the app is accessed by user) to elapse before the app is
+            auto deleted, e.g. "2 hours, 2h".  
+            A value of Inf means no auto delete.
```

Comment:
that's good point...we should probably add in `get`...

`patch` is intentionally left out to keep this PR simple...we do plan to add it later tho

Reply:
I will use follow up few PRs to support get/update/list to make it easier to review and write. 
(there is no UpdateApp api, supporting this would add couple hundreds line of code...)